pkg/apiserver/webhooks/populators-validate_test.go

```diff
@@ -145,6 +145,58 @@ var _ = Describe("Validating Webhook", func() {
 			Expect(resp.Allowed).To(BeFalse())
 		})
 
+		It("should VolumeImportSource with incomplete VDDK source", func() {
+			source := &cdiv1.ImportSourceType{
+				VDDK: &cdiv1.DataVolumeSourceVDDK{
+					BackingFile: "",
+					URL:         "",
+					UUID:        "",
+					Thumbprint:  "",
+					SecretRef:   "",
+				},
+			}
+			importCR := newVolumeImportSource(cdiv1.DataVolumeKubeVirt, source)
+			resp := validateVolumeImportSourceCreate(importCR)
+			Expect(resp.Allowed).To(BeFalse())
+		})
+
+		It("should reject multi-stage VolumeImportSource without TargetClaim", func() {
+			source := &cdiv1.ImportSourceType{
+				VDDK: &cdiv1.DataVolumeSourceVDDK{
+					BackingFile: "[iSCSI_Datastore] vm/vm_1.vmdk",
+					URL:         "https://vcenter.corp.com",
+					UUID:        "52260566-b032-36cb-55b1-79bf29e30490",
+					Thumbprint:  "20:6C:8A:5D:44:40:B3:79:4B:28:EA:76:13:60:90:6E:49:D9:D9:A3",
+					SecretRef:   "vddk-credentials",
+				},
+			}
+			importCR := newVolumeImportSource(cdiv1.DataVolumeKubeVirt, source)
+			importCR.Spec.Checkpoints = []cdiv1.DataVolumeCheckpoint{
+				{Current: "test", Previous: ""},
+			}
+			resp := validateVolumeImportSourceCreate(importCR)
+			Expect(resp.Allowed).To(BeFalse())
+		})
+
+		It("should accept multi-stage VolumeImportSource with TargetClaim", func() {
```

Comment:
what about test that accepts update of multi-stage, and no update of targetclaim